src/lib/core/fiber.h

```diff
@@ -1178,8 +1199,6 @@ fiber_cxx_invoke(fiber_func f, va_list ap)
 	}
 }
 
-#endif /* defined(__cplusplus) */
-
 static inline void *
 region_aligned_alloc_cb(void *ctx, size_t size)
```

Comment:
Looks like `region_aligned_alloc_cb` is never used so instead of hiding it from C code, better remove it.

Reply:
Removed.